test/test_model.py

```diff
@@ -103,7 +107,8 @@ def check_waveform(assertion, generated_waveform):
 
     # Expect most of the power to be at the 3 frequencies we trained
     # on.
-    assertion(expected_power, 0.9 * power_sum)
+    assertion(expected_power, 0.75 * power_sum)
```

Comment:
As I commented there, I think that my formula for receptive field is correct. And even if it wasn't, I still think that this test is wrong, because it should check if the model can overfit on the training data, so if the training data doesn't pass the test, how can we expect output of a network which should exactly reproduce the training data to pass it?